server/src/main/java/io/littlehorse/common/model/getable/global/wfspec/node/EdgeModel.java

```diff
@@ -149,9 +149,10 @@ public void mutateVariables(ThreadRunModel threadRun, Optional<VariableValueMode
                 //
                 // The only thing we do here is add more info to the exception so that
                 // it is easier for the user to debug why their WfRun failed.
-                exn.addPrefix("Mutating variable %s with operation %s"
-                        .formatted(mutation.getLhsName(), mutation.getOperation()));
-                throw exn;
+                throw new LHVarSubError(
+                        exn,
```

Comment:
I created an e2e to verify this behavior, you are right the message is different in this branch, thanks!. However, I think master branch is also incorrect because the exception message is appended twice in the exception message, for example:

`Failed evaluating edge with sink node 1-print-number-TASK: Mutating variable value-a with operation DIVIDE: Caught unexpected error when mutating variables: / by zero / by zero`

Reply:
I see. The proper message would probably be:

```
Failed evaluating edge with sink node 1-print-number-TASK: Mutating variable value-a with operation DIVIDE: Caught unexpected error when mutating variables: / by zero
```

Reply:
I added an e2e that verifies that the actual message contains `/ by zero` as a message, so we keep compatibility between versions. Also, I added a unit test that verifies the output message for `LHVarSubError` exceptions